src/elements/content-uploader/UploadsManager.scss

```diff
@@ -3,6 +3,7 @@
 .be.bcu-uploads-manager-container {
     position: fixed;
     bottom: 0;
+    display: none;
     width: 100%;
     max-height: 0;
     transition: max-height .5s;
```

Comment:
there's a `transition` on this class, does it still work? I think there are conflicts when trying to animate and update the `display` property at the same time (I think some issues with `visibility` too)

Reply:
@tjuanitas It still works for me on Chrome and Safari

https://github.com/box/box-ui-elements/assets/141409011/bd384bcc-ed16-4b11-9008-4ced47d52f3d

Reply:
that transition occurs when toggling `bcu-is-expanded` which happens after `display: initial` is already applied. you'll want to test toggling `bcu-is-visible` which handles the transition between `display: none` -> `display: initial`. this will happen on drag-or-drop or when a file is uploaded

https://github.com/box/box-ui-elements/assets/7311041/c3e53f70-e1f8-4afc-8b53-b9677e0798de